autode/neb/original.py

```diff
@@ -426,18 +444,81 @@ def from_file(
 
             # TODO: test reasonableness of this function...
             # use a shifted tanh to interpolate in [0.005, 0.2005]
-            k = (
+            init_k = ForceConstant(
                 0.1 * (np.tanh((max_de.to("kcal mol-1") - 40) / 20) + 1)
-                + 0.005
+                + 0.005,
+                units="Ha / Å^2",
             )
 
-        if k is None:  # choose a sensible default
-            k = 0.1
+        if init_k is None:  # choose a sensible default
+            init_k = 0.1
 
-        logger.info(f"Using k = {k:.6f} Ha Å^-1 as the NEB force constant")
-        return cls(species_list=molecules, k=k)
+        logger.info(
+            f"Using k = {init_k:.6f} Ha Å^-1 as the NEB force constant"
+        )
+        return cls.from_list(molecules, init_k=init_k)
 
-    def _minimise(self, method, n_cores, etol, max_n=30) -> None:
+    @classmethod
+    def from_list(
+        cls,
+        species_list: List[Species],
+        init_k: ForceConstant = ForceConstant(0.1, units="Ha / Å^2"),
+    ) -> "NEB":
+        """
+        Nudged elastic band constructed from list of species
+
+        -----------------------------------------------------------------------
+        Arguments:
+            species_list: Full set of initial images that will form the while NEB
+
+            init_k: Force constant
+
+        Returns:
+            (NEB):
+        """
+        neb = cls(init_k=init_k)
+
+        for species in species_list:
+            neb.images.append_species(species)
+
+        logger.info(f"Initialised a NEB with {len(neb.images)} images")
+        return neb
+
+    @classmethod
+    def from_end_points(
+        cls,
+        initial: Species,
+        final: Species,
+        num: int,
+        init_k: ForceConstant = ForceConstant(0.1, units="Ha / Å^2"),
+    ) -> "NEB":
+        """
+        Construct a nudged elastic band from only the endpoints. The atomic
+        ordering must be identical in the initial and final species
```

Comment:
I think it would be better to have a sanity check function that would check the ordering of atoms and stops if they are different. For example, I tried to write a sanity check function for the dhs method (the new one that i am still working on).

https://github.com/shoubhikraj/autodE/blob/db3ebc9606fed3a37b0958981d03178ca300ff84/autode/bracket/imagepair.py#L148-L182